server/src/main/java/io/crate/analyze/AnalyzedColumnDefinition.java

```diff
@@ -92,6 +92,7 @@
 
     private boolean isPrimaryKey = false;
     private boolean isNotNull = false;
+    private boolean docValuesSpecifiedAndDisabled = false;
```

Comment:
Could either make this a `Boolean docValues` to use `null` for "not declared; use-default". Or change it do `boolean docvalues` and apply the default value by using `StorageSupport.docValuesDefault` if not declared by the user.

Reply:
Went with the second approach. Thanks for pointing out `StorageSupport.docValuesDefault` - missed that in the previous iteration. Addressed this and another related comment in https://github.com/crate/crate/pull/13238/commits/ba077b7b5c33d654cf73078e509bf03c64dac28e